CheckSources.ps1

```diff
@@ -0,0 +1,70 @@
+﻿[CmdletBinding()]
+param(
+    [Parameter(Mandatory = $false)]
+    [string]$VersionsFilter = "*",
+    [Parameter(Mandatory = $true)]
+    [ValidateScript( {Test-Path $_ -PathType 'Container'})] 
+    [string]$InstallSourcePath
+)
+
+function Find-SitecoreVersions {
```

Comment:
I'll merge this PR but I'll create an issue to refactor the whole Build.ps1 file.

Instead of creating a brand new file with an "almost" copy of Find-SitecoreVersions, we should modify the Find-SitecoreVersions function in Build.ps1 script to do just that, simply find the versions. 

Would be another breaking change - but it's still early on in the adoption of the source. 

Best approach is probably to remove Build.ps1, create a PSM1 file and follow PS best practices for creating Approved Verb Functions?

We can revisit.